app/services/hyrax/custom_queries/find_file_metadata.rb

```diff
@@ -9,7 +9,8 @@ class FindFileMetadata
       def self.queries
         [:find_file_metadata_by,
          :find_file_metadata_by_alternate_identifier,
-         :find_many_file_metadata_by_ids]
+         :find_many_file_metadata_by_ids,
+         :find_many_file_metadata_by_use]
```

Comment:
That was a nice catch. I'm surprised the tests passed.  I thought queries had to be registered to be used through custom_queries.

Reply:
seems like they worked okay with the wings adapter, but not the memory adapter (?)

not sure why the wings one worked okay, but this is the first time this gets used with a different adapter.